ui/e2e/src/config/base.playwright.config.ts

```diff
@@ -31,8 +31,16 @@ const config: PlaywrightTestConfig = {
     timeout: 5000,
   },
 
-  /* Run tests in files in parallel */
-  fullyParallel: true,
+  /**
+   * Do not run tests in files in parallel. This makes it possible to have tests
+   * within a given file mutate the same dashboard and reset after each test.
+   * If the tests ran in parallel, this would lead to inconsistencies.
+   *
+   * TODO: figure out how to use the create/delete endpoints, so we can use
+   * unique dashboards per test. These will allow us to parallelize these tests
+   * again.
```

Comment:
These endpoints exist, but I couldn't get `create` to work and failed at debugging the backend issues on my own. I didn't want to block the incremental test improvement on getting help with that, so this is a stepping stone. Hopefully, I can improve it in the near future.

Reply:
I found the issue and fixed it with #923. You should be good now :)

Reply:
Thanks! Adjusted to use the create/delete endpoints and adjusted the setting back to run in parallel.